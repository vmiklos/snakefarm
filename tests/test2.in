debug 1
out 1
time 100
load test2.map
addplayer 1
addsnake 1 1 0 1 1 0
6 0
endsnake
addplayer 2
addsnake 2 2 0 1 1 0
11 0
endsnake
comment # a proto implementációjától függõen a nem falnak ütközõ kígyó lehet, hogy lép a játék vége elõtt, és lehet, hogy nem lép
step
exit