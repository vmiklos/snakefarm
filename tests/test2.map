1;W;2;5
2;W;3;6
3;W;4;7
4;W;0;8

5;W;6;9
6;0;7;10
7;0;8;11
8;W;0;12

9;W;10;13
10;0;11;14
11;0;12;15
12;W;0;16

13;W;14;0
14;W;15;0
15;W;16;0
16;W;0;0