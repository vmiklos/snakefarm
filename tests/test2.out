Event MapLoad test2.map
# a proto implementációjától függõen a nem falnak ütközõ kígyó lehet, hogy lép a játék vége elõtt, és lehet, hogy nem lép
Event Step 1
StepEvent Snake_Wall
StepEvent SnakeDie
End Step
Event Win 2
Event Exit